### PR TITLE
Fixes #1114: Implemented small description for each query

### LINKED ISSUE #1114: Implement small query description for each query in Infobox.
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**
Currently Infobox do not show a one line description for search query.
**Expected behavior:**
Implement small query description for each query in Infobox similar to market leader Google.
**Screenshots of the issue:**

image

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1114

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- Implemented small description for each query similar to Market Leader Google. I have implemented it using Wikidata Open Source API.

- https://pr-1115-fossasia-susper.surge.sh

image

### CODE REVIEW

File: src/app/services/knowledgeapi.service.ts [hunk 1]
```diff
@@ -55,6 +56,21 @@ export class KnowledgeapiService {
       res.json()
       ).catch(this.handleError);
     }
+  getQueryDescription(searchquery) {
+    let params = new URLSearchParams();
```
Comment (simsausaurabh) [inline src/app/services/knowledgeapi.service.ts]: Use ```const``` if the value is never assigned again.

File: src/app/services/knowledgeapi.service.ts [hunk 2]
```diff
@@ -55,6 +56,21 @@ export class KnowledgeapiService {
...
+
+    params.set('origin', '*');
+    params.set('action', 'wbsearchentities');
+    params.set('search', searchquery);
+    params.set('language', 'en');
+    params.set('format', 'json');
+    let headers = new Headers({ 'Accept': 'application/json' });
```
Comment (simsausaurabh) [inline src/app/services/knowledgeapi.service.ts]: Use ```const``` if the value is never assigned again.

File: src/app/services/knowledgeapi.service.ts [hunk 3]
```diff
@@ -55,6 +56,21 @@ export class KnowledgeapiService {
...
+    let options = new RequestOptions({ headers: headers, search: params });
```
Comment (simsausaurabh) [inline src/app/services/knowledgeapi.service.ts]: Use ```const``` if the value is never assigned again.

**Comment by praveenojha33** [comment]
@simsausaurabh I have made all suggested changes. Please review again.
image

**Comment by mariobehling** [comment]
Looking at the picture of "Trump" do you see that you squeeze that image? With one fix somewhere another area is affected and a new issue appears. It does not make sense to work in that way - fix one thing and break another.

**Comment by praveenojha33** [comment]
@mariobehling I have not made any changes to image (just made some adjustment to position of image). But if you think there are issues with image (stretching or squeezing) I have opened an issue for the same https://github.com/fossasia/susper.com/issues/1116 .
Please compare the images 
On susper.com
image

In this PR:
image

**Pull request merged**